modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-15-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-15-0_{context}"]
+= Release notes for {gitops-title} 1.15.0
+
+{gitops-title} 1.15.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
```

Comment:
Should we mention `4.17` as well ?

Reply:
Thanks for the comment, @anandf. As we spoke on Slack, I have gone ahead and made this change in a different PR. You can refer to the change [here](https://84086--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes-1-14.html#GitOps-compatibility-support-matrix_gitops-release-notes).

Reply:
But that was for 1.14.0 IIUC?